sbtplugin/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -52,7 +52,7 @@ object ScalaNativePluginInternal {
   }
 
   lazy val projectSettings = Seq(
-    addCompilerPlugin("org.scala-native" %% "nscplugin" % "0.1-SNAPSHOT"),
+    addCompilerPlugin("org.scala-native" %% "nscplugin" % "0.1-SNAPSHOT" cross CrossVersion.full),
```

Comment:
It makes me sense to write `%` instead of `%%` when using `cross`. The latter overrides the meaning of `%%`.